products/nexus.md

```diff
@@ -2,42 +2,279 @@
+    releaseDate: 2022-07-27
+    support: 2023-08-03
+    eol: 2024-01-27
+    latest: "3.41.1"
+    latestReleaseDate: 2022-06-24
+
+-   releaseCycle: "3.40"
+    releaseDate: 2022-06-22
+    support: 2023-08-03
+    eol: 2023-12-22
+    latest: "3.40.1"
+    latestReleaseDate: 2022-06-24
+
+-   releaseCycle: "3.39"
+    releaseDate: 2022-05-23
+    support: 2023-08-03
+    eol: 2023-11-19
+    latest: "3.39.0"
+    latestReleaseDate: 2022-05-23
+
+-   releaseCycle: "3.38"
+    releaseDate: 2022-03-03
+    support: 2023-08-03
+    eol: 2023-09-25
+    latest: "3.38.1"
+    latestReleaseDate: 2022-03-29
+
+-   releaseCycle: "3.37"
+    releaseDate: 2021-11-14
+    support: false
+    eol: 2023-05-24
+    latest: "3.37.3"
+    latestReleaseDate: 2021-12-29
+
+-   releaseCycle: "3.36"
+    releaseDate: 2021-10-29
+    support: false
+    eol: 2023-04-29
+    latest: "3.36.0"
+    latestReleaseDate: 2021-10-29
+
+-   releaseCycle: "3.35"
+    releaseDate: 2021-10-13
+    support: false
+    eol: 2023-04-13
+    latest: "3.35.0"
+    latestReleaseDate: 2021-10-13
+
+-   releaseCycle: "3.34"
+    releaseDate: 2021-08-31
+    support: false
+    eol: 2023-03-03
+    latest: "3.34.1"
+    latestReleaseDate: 2021-09-22
+
+-   releaseCycle: "3.33"
+    releaseDate: 2021-08-04
+    support: false
+    eol: 2023-02-04
+    latest: "3.33.1"
+    latestReleaseDate: 2021-08-17
+
+-   releaseCycle: "3.32"
+    releaseDate: 2021-07-06
+    support: false
+    eol: 2023-01-08
+    latest: "3.32.1"
+    latestReleaseDate: 2021-12-20
+
+-   releaseCycle: "3.30"
+    releaseDate: 2021-03-04
+    support: false
+    eol: 2022-09-04
+    latest: "3.30.1"
+    latestReleaseDate: 2021-04-22
+
+-   releaseCycle: "3.29"
+    releaseDate: 2020-12-04
+    support: false
+    eol: 2022-06-04
+    latest: "3.29.2"
+    latestReleaseDate: 2021-01-06
 
 ---
 
 > [Nexus Repository OSS](https://help.sonatype.com/repomanager3/product-information/download) is an open
 > source repository manager developed by Sonatype that supports many artifact formats, including
 > Docker, Java, and npm.
 
-Only the latest version of each major release is supported.
+## [Versioning Scheme](https://help.sonatype.com/docs/sonatype-sunsetting-information/sonatype-nexus-repository-3-versions-status)
+
+- Sonatype Nexus Repository releases are considered generally available and fully supported for a term of **1 year after the version's release date**.
+- At that point, Sonatype provides extended maintenance for each Sonatype Nexus Repository version for an additional **6 months** before that version is considered sunset [eol].
+- For versions requiring patch releases, the full version line (e.g., 3.53.0 - 3.53.1, 3.47.0 - 3.47.1) will use the dates of the initial major release (e.g., 3.53.0 or 3.47.0) to determine when the versions will move through the PDLC process.
+
+Get more information on [Sonatype Sunsetting](https://help.sonatype.com/docs/sonatype-sunsetting-information).
```

Comment:
Those information are for the commercial version while this page document the OSS versions. Is it true for the OSS version ?